platform-sdk/docs/proposals/TSS-Encryption-Key-Automation/TSS-Encryption-Key-Automation.md

```diff
@@ -198,16 +194,17 @@ No new configuration is needed.
 
 A health metric should be created to indicate the state the `TssBaseService` is in:
 
-1. distributing the public `tssEncryptionKey` to the network.
-2. public `tssEncryptionKey` is good, waiting for the next candidate roster to be keyed and adopted.
-3. Able to participate in signing with the active roster. (Private shares are decryptable with current
-   `tssEncryptionKey`.)
+1. State 1: distributing the public `tssEncryptionKey` to the network.
+2. State 2: the latest `tssEncryptionKey` is in the state, waiting to receive private shares.
+3. State 3: Private shares exist for this node in the latest `active roster` and the `TssBaseService` can execute
+   normally.
```

Comment:
Minor: "shares exist ... in ... roster" sounds a bit confusing because the roster doesn't actually contain any shares. I suggest to rephrase this for clarity and perhaps refer to a specific store/state where the private shares are actually stored by the TSS service.